File: d09Fri/TypeScript/program/p00check_id/checkId.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
class CheckId {
    constructor(id, pw) {
        this._id = id;
        this._pw = pw;
    }
    /* public set id(value: string){
      this._id = value;
    }
    public get id(){
      return this._id;
    }
    public set pw(value: string){
      this._pw = value;
    }
    public get pw(){
      return this._pw;
    } */
    checkId() {
        if (this._id === "jin" && this._pw === "111") {
            console.log("yep");
        }
        else {
            console.log("nope");
        }
    }
}
exports.default = CheckId;

File: d09Fri/C/program/p00check_id/checkId.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

struct CheckId{
  char *id;
  char *pw;
};

void checkId(struct CheckId ckd){
  int id = strcmp(ckd.id, "jin");
  int pw = strcmp(ckd.pw, "111");
  if(id == 0 && pw == 0) puts("yey");
  else puts("nope");
}